### PR TITLE
Migrate adaptive packages to NNBD

### PR DESCRIPTION
**Pull request by guidezpl**
Migrates `adaptive_navigation` and `adaptive_breakpoints` (new pre-1.0.0 versions)
Prepares `adaptive_components` for initial release on pub.dev (0.0.1)

### CODE REVIEW

File: adaptive_navigation/pubspec.yaml [hunk 1]
```diff
@@ -4,12 +4,14 @@ version: 0.0.1
...
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  adaptive_breakpoints: ^0.0.2
+  adaptive_breakpoints:
+    # TODO: Use pub dependency once NNBD beta is released.
+    path: ../adaptive_breakpoints
```
Comment (rami-a) [inline adaptive_navigation/pubspec.yaml]: Will this actually work if someone is using the `adaptive_navigation` pub package?

**Review by rami-a — APPROVED** [review]
LGTM, though should the version number be bumped anywhere?

**Comment by guidezpl** [comment]
Should've created a draft PR, it's not ready to be merged

**Review by JoseAlba — APPROVED** [review]
```
██╗      ██████╗ ████████╗███╗   ███╗
██║     ██╔════╝ ╚══██╔══╝████╗ ████║
██║     ██║  ███╗   ██║   ██╔████╔██║
██║     ██║   ██║   ██║   ██║╚██╔╝██║
███████╗╚██████╔╝   ██║   ██║ ╚═╝ ██║
╚══════╝ ╚═════╝    ╚═╝   ╚═╝     ╚═╝
```

File: adaptive_breakpoints/example/pubspec.yaml [hunk 2]
```diff
@@ -3,8 +3,10 @@ description: A new Flutter project.
 
 version: 0.0.1
 
+publish_to: 'none'
+
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
```
Comment (rami-a) [inline adaptive_breakpoints/example/pubspec.yaml]: Why is this not 2.12?

File: adaptive_components/example/pubspec.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ description: A new Flutter project.
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
```
Comment (rami-a) [inline adaptive_components/example/pubspec.yaml]: Should this be 2.12?

File: adaptive_navigation/example/pubspec.yaml [hunk 4]
```diff
@@ -6,18 +6,18 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
```
Comment (rami-a) [inline adaptive_navigation/example/pubspec.yaml]: Same here

**Review by rami-a — APPROVED** [review]

File: adaptive_breakpoints/example/pubspec.yaml (hunk 2 above)
Comment (guidezpl) [reply adaptive_breakpoints/example/pubspec.yaml]: Fixed

**Pull request merged**